*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:38:05 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:34056 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x1dc3a00
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 9825)
spawned uWSGI worker 1 (pid: 9826, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 9827)
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:38:13 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8000).
bind(): Address already in use [core/socket.c line 769]
--- no python application found, check your startup logs for errors ---
[pid: 9826|app: -1|req: -1/1] 127.0.0.1 () {38 vars in 577 bytes} [Thu Mar 12 21:38:27 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 9826|app: -1|req: -1/2] 127.0.0.1 () {38 vars in 577 bytes} [Thu Mar 12 21:38:28 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 9826|app: -1|req: -1/3] 127.0.0.1 () {38 vars in 577 bytes} [Thu Mar 12 21:39:28 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 9826|app: -1|req: -1/4] 127.0.0.1 () {36 vars in 559 bytes} [Thu Mar 12 21:39:31 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
gateway "uWSGI http 1" has been buried (pid: 9827)
...brutally killing workers...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ssgll/Desktop/web/linesys
closing all non-uwsgi socket fds > 2 (max_fd = 4096)...
found fd 3 mapped to socket 0 (127.0.0.1:34056)
running /home/ssgll/.local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:45:00 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/ssgll/Desktop/web/linesys
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 5
uwsgi socket 0 bound to TCP address 127.0.0.1:45243 (port auto-assigned) fd 4
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x2028800
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 9825)
spawned uWSGI worker 1 (pid: 10127, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 10128)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 10128)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:50:41 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:37679 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x1e40a40
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 10588)
spawned uWSGI worker 1 (pid: 10590, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 10591)
--- no python application found, check your startup logs for errors ---
[pid: 10590|app: -1|req: -1/1] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 21:50:52 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 10591)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:55:22 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:35645 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x27e5a40
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 11776)
spawned uWSGI worker 1 (pid: 11778, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 11779)
--- no python application found, check your startup logs for errors ---
[pid: 11778|app: -1|req: -1/1] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 21:56:34 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 11779)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 21:58:26 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:39962 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x268da10
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 11909)
spawned uWSGI worker 1 (pid: 11911, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 11912)
--- no python application found, check your startup logs for errors ---
[pid: 11911|app: -1|req: -1/1] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 21:58:43 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 11911|app: -1|req: -1/2] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 21:58:45 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 11911|app: -1|req: -1/3] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 21:58:46 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 11911|app: -1|req: -1/4] 127.0.0.1 () {36 vars in 547 bytes} [Thu Mar 12 21:58:51 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 11912)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 22:07:09 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /linesys
chdir(): No such file or directory [core/uwsgi.c line 2623]
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 22:07:40 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:38202 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x23aaa40
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from flask_script import Manager
ModuleNotFoundError: No module named 'flask_script'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 12384)
spawned uWSGI worker 1 (pid: 12386, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 12387)
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 22:24:21 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 7145
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8000).
bind(): Address already in use [core/socket.c line 769]
--- no python application found, check your startup logs for errors ---
[pid: 12386|app: -1|req: -1/1] 127.0.0.1 () {36 vars in 547 bytes} [Thu Mar 12 22:24:35 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12386|app: -1|req: -1/2] 127.0.0.1 () {36 vars in 559 bytes} [Thu Mar 12 22:24:47 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 12387)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 22:44:56 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:41704 (port auto-assigned) fd 3
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x18f7a50
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
unable to find "application" callable in file manage.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 14340)
spawned uWSGI worker 1 (pid: 14342, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 14343)
--- no python application found, check your startup logs for errors ---
[pid: 14342|app: -1|req: -1/1] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 22:45:04 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14342|app: -1|req: -1/2] 127.0.0.1 () {38 vars in 590 bytes} [Thu Mar 12 22:45:06 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 14343)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 23:09:03 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:37183 (port auto-assigned) fd 3
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x1cd9a70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
unable to find "application" callable in file manage.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15524)
spawned uWSGI worker 1 (pid: 15526, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 15527)
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 23:17:10 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8000).
bind(): Address already in use [core/socket.c line 769]
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 15527)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 23:17:56 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:44972 (port auto-assigned) fd 3
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x163bb50
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
added /home/ssgll/virtualenvdir/webenv/ to pythonpath.
unable to find "application" callable in file manage.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15740)
spawned uWSGI worker 1 (pid: 15742, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 15744)
--- no python application found, check your startup logs for errors ---
[pid: 15742|app: -1|req: -1/1] 127.0.0.1 () {36 vars in 546 bytes} [Thu Mar 12 23:18:05 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 15742|app: -1|req: -1/2] 127.0.0.1 () {36 vars in 558 bytes} [Thu Mar 12 23:18:09 2020] GET /index/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 15744)
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Thu Mar 12 23:21:39 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:36851 (port auto-assigned) fd 3
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0xac1b50
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
added /home/ssgll/virtualenvdir/webenv/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xac1b50 pid: 15934 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15934)
spawned uWSGI worker 1 (pid: 15936, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 15938)
[pid: 15936|app: 0|req: 1/1] 127.0.0.1 () {36 vars in 547 bytes} [Thu Mar 12 23:21:54 2020] GET / => generated 232 bytes in 5 msecs (HTTP/1.1 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 2/2] 127.0.0.1 () {36 vars in 559 bytes} [Thu Mar 12 23:21:57 2020] GET /index/ => generated 30 bytes in 1129 msecs (HTTP/1.1 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 3/3] 127.0.0.1 () {38 vars in 573 bytes} [Fri Mar 13 00:28:11 2020] GET / => generated 232 bytes in 1542 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 4/4] 127.0.0.1 () {36 vars in 552 bytes} [Fri Mar 13 00:28:17 2020] GET /index => generated 263 bytes in 1217 msecs (HTTP/1.0 308) 3 headers in 136 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 5/5] 127.0.0.1 () {36 vars in 554 bytes} [Fri Mar 13 00:28:18 2020] GET /index/ => generated 30 bytes in 414 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 6/6] 127.0.0.1 () {36 vars in 559 bytes} [Fri Mar 13 00:28:49 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 7/7] 127.0.0.1 () {34 vars in 485 bytes} [Fri Mar 13 00:28:49 2020] GET /favicon.ico => generated 232 bytes in 0 msecs (HTTP/1.1 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 8/8] 127.0.0.1 () {40 vars in 703 bytes} [Fri Mar 13 00:35:59 2020] GET / => generated 232 bytes in 1 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 9/9] 127.0.0.1 () {38 vars in 617 bytes} [Fri Mar 13 00:35:59 2020] GET /favicon.ico => generated 232 bytes in 0 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 10/10] 127.0.0.1 () {40 vars in 703 bytes} [Fri Mar 13 00:36:01 2020] GET / => generated 232 bytes in 0 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 11/11] 127.0.0.1 () {38 vars in 682 bytes} [Fri Mar 13 00:36:06 2020] GET /index => generated 263 bytes in 1 msecs (HTTP/1.0 308) 3 headers in 136 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 12/12] 127.0.0.1 () {38 vars in 684 bytes} [Fri Mar 13 00:36:06 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 13/13] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:37:06 2020] GET /index/ => generated 30 bytes in 14 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 14/14] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:37:08 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 15/15] 127.0.0.1 () {38 vars in 672 bytes} [Fri Mar 13 00:38:52 2020] GET / => generated 232 bytes in 1 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 16/16] 127.0.0.1 () {38 vars in 617 bytes} [Fri Mar 13 00:38:52 2020] GET /favicon.ico => generated 232 bytes in 0 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 17/17] 127.0.0.1 () {38 vars in 684 bytes} [Fri Mar 13 00:38:55 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 18/18] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:39:15 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 19/19] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:40:14 2020] GET /index/ => generated 30 bytes in 0 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 20/20] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:40:32 2020] GET /index/ => generated 30 bytes in 0 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 21/21] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:40:32 2020] GET /index/ => generated 30 bytes in 0 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 22/22] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:40:35 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 23/23] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:44:46 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 24/24] 127.0.0.1 () {38 vars in 684 bytes} [Fri Mar 13 00:54:46 2020] GET /index/ => generated 30 bytes in 2 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 25/25] 127.0.0.1 () {38 vars in 623 bytes} [Fri Mar 13 00:54:46 2020] GET /favicon.ico => generated 232 bytes in 0 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 26/26] 127.0.0.1 () {38 vars in 684 bytes} [Fri Mar 13 00:56:04 2020] GET /index/ => generated 30 bytes in 2 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 15936|app: 0|req: 27/27] 127.0.0.1 () {40 vars in 715 bytes} [Fri Mar 13 00:58:06 2020] GET /index/ => generated 30 bytes in 3 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 15936|app: 0|req: 28/28] 127.0.0.1 () {38 vars in 618 bytes} [Fri Mar 13 00:58:06 2020] GET /favicon.ico => generated 232 bytes in 0 msecs (HTTP/1.0 404) 2 headers in 87 bytes (1 switches on core 1)
Gracefully killing worker 1 (pid: 15936)...
gateway "uWSGI http 1" has been buried (pid: 15938)
...brutally killing workers...
gateway 1 (uWSGI http 1, pid: 15938) annihilated
worker 1 buried after 10 seconds
binary reloading uWSGI...
chdir() to /home/ssgll/Desktop/web/linesys
closing all non-uwsgi socket fds > 2 (max_fd = 4096)...
found fd 3 mapped to socket 0 (127.0.0.1:36851)
running /home/ssgll/.local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 13 00:58:54 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 4096
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 5
uwsgi socket 0 bound to TCP address 127.0.0.1:45247 (port auto-assigned) fd 4
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x166b970
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
added /home/ssgll/virtualenvdir/webenv/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 34 seconds on interpreter 0x166b970 pid: 15934 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 15934)
spawned uWSGI worker 1 (pid: 20418, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 20419)
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 13 01:44:53 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
writing pidfile to ./uwsgi.pid
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:43719 (port auto-assigned) fd 3
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x1fff480
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
added /home/ssgll/virtualenvdir/webenv/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 15 seconds on interpreter 0x1fff480 pid: 3437 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3437)
spawned uWSGI worker 1 (pid: 3449, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 3450)
[pid: 3449|app: 0|req: 2/2] 127.0.0.1 () {40 vars in 727 bytes} [Fri Mar 13 01:45:14 2020] GET /index/ => generated 30 bytes in 816 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 3449|app: 0|req: 2/2] 127.0.0.1 () {40 vars in 727 bytes} [Fri Mar 13 01:45:15 2020] GET /index/ => generated 30 bytes in 394 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
[pid: 3449|app: 0|req: 3/3] 127.0.0.1 () {40 vars in 727 bytes} [Fri Mar 13 01:45:15 2020] GET /index/ => generated 30 bytes in 0 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 0)
[pid: 3449|app: 0|req: 4/4] 127.0.0.1 () {40 vars in 727 bytes} [Fri Mar 13 01:46:00 2020] GET /index/ => generated 30 bytes in 1 msecs (HTTP/1.0 200) 2 headers in 79 bytes (1 switches on core 1)
gateway "uWSGI http 1" has been buried (pid: 3450)
...gracefully killing workers...
worker 1 buried after 18 seconds
binary reloading uWSGI...
chdir() to /home/ssgll/Desktop/web/linesys
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (127.0.0.1:43719)
running /home/ssgll/.local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 13 02:31:11 2020] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-39) on 12 March 2020 12:05:14
os: Linux-3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /home/ssgll/Desktop/web/linesys
detected binary path: /home/ssgll/.local/bin/uwsgi
chdir() to /home/ssgll/Desktop/web/linesys
your processes number limit is 4096
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 5
uwsgi socket 0 bound to TCP address 127.0.0.1:32939 (port auto-assigned) fd 4
Python version: 3.7.6 (default, Mar 12 2020, 07:18:52)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
Python main interpreter initialized at 0x2163340
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 166752 bytes (162 KB) for 2 cores
*** Operational MODE: threaded ***
added /home/ssgll/virtualenvdir/webenv/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0x2163340 pid: 3437 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 3437)
spawned uWSGI worker 1 (pid: 4317, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 12 ***
spawned uWSGI http 1 (pid: 4318)
